app/views/notes/show.html.erb

```diff
@@ -86,6 +86,10 @@
     </table>
   <% end %>
 
+  <% if current_user && ( current_user.role == "admin" || current_user.role == "moderator") && !(@node.has_tag('locked')) %>
```

Comment:
What if, in follow up, we moved this button into the advanced tagging menu? This would then appear on wikis too!